cmd/controller/app/start_test.go

```diff
@@ -85,6 +85,15 @@ func TestFlagsAndConfigFile(t *testing.T) {
 	}
 
 	tests := []testCase{
+		{
+			yaml: ``,
+			args: func(tempFilePath string) []string {
+				return []string{"--kubeconfig=valid"}
+			},
+			expConfig: configFromDefaults(func(tempDir string, cc *config.ControllerConfiguration) {
+				cc.KubeConfig = "valid"
+			}),
+		},
```

Comment:
I don't understand how this new test is related to the change in the logging options parsing?

Reply:
This is a test-case that was missing, we were not testing the case in which there is no config file that is loaded.